gravitee-apim-console-webui/src/entities/management-api-v2/audit/auditResponse.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Audit } from './audit';
+
+import { Links } from '../links';
+import { Pagination } from '../pagination';
+
+export interface SearchApiAuditParam {
+  page?: number;
+  perPage?: number;
+  from?: number;
+  to?: number;
+  events?: string;
+}
+
+export interface AuditsResponse {
```

Comment:
I wonder if we couldn't use `PagedResult<Audit>` WDYT ?

Reply:
If it matches to the OpenAPI of the RestV2 I can. I will take a look

Reply:
Normally yes : gravitee-apim-console-webui/src/entities/management-api-v2/pagedResult.ts